db/seeds.rb

```diff
@@ -1,7 +1,89 @@
+   { name: 'garlic' },
+   { name: 'chilli' },
+   { name: 'prawns' },
+   { name: 'gujarati masala' },
+   { name: 'garam masala' },
+   { name: 'yogurth' },
+   { name: 'coriander' },
+   { name: 'steak' },
+   { name: 'oregano' },
+   { name: 'red wine' },
+   { name: 'lemon' }].each{|x| Ingredient.create(x)}
+
+lorem = "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Proin nibh augue, suscipit a, scelerisque sed, lacinia in, mi. Cras vel lorem.
+  Etiam pellentesque aliquet tellus. Phasellus pharetra nulla ac diam. Quisque semper justo at risus. Donec venenatis, turpis vel hendrerit interdum,
+  dui ligula ultricies purus, sed posuere libero dui id orci. Nam congue, pede vitae dapibus aliquet, elit magna vulputate arcu, vel tempus metus leo non est.
+  Etiam sit amet lectus quis est congue mollis. Phasellus congue lacus eget neque. Phasellus ornare, ante vitae consectetuer consequat,
+  purus sapien ultricies dolor, et mollis pede metus eget nisi. Praesent sodales velit quis augue. Cras suscipit, urna at aliquam rhoncus,
+  urna quam viverra nisi, in interdum massa nibh nec erat."
+
+[
+   { name: 'venetian-style pasta', description: lorem },
+   { name: 'tomato and thyme cod', description:lorem },
+   { name: 'lemon chicken with spring veg noodles', description: lorem },
+   { name: 'curry in a hurry', description:lorem },
+   { name: '20-minutes beef in red wine', description: lorem }
+].each{|x| Recipe.create(x)}
+
+IngredientsRecipe.create[
+  {recipe_id: 1, ingredient_id: 1},
+  {recipe_id: 1, ingredient_id: 2},
+  {recipe_id: 1, ingredient_id: 3},
+  {recipe_id: 1, ingredient_id: 4},
+  {recipe_id: 1, ingredient_id: 5},
+  {recipe_id: 1, ingredient_id: 6},
+  {recipe_id: 1, ingredient_id: 7},
+  {recipe_id: 1, ingredient_id: 8},
+  {recipe_id: 2, ingredient_id: 1},
+  {recipe_id: 2, ingredient_id: 2},
+  {recipe_id: 2, ingredient_id: 9},
+  {recipe_id: 2, ingredient_id: 10},
+  {recipe_id: 2, ingredient_id: 11},
+  {recipe_id: 2, ingredient_id: 12},
+  {recipe_id: 2, ingredient_id: 13},
+  {recipe_id: 3, ingredient_id: 14},
+  {recipe_id: 3, ingredient_id: 15},
+  {recipe_id: 3, ingredient_id: 32},
+  {recipe_id: 3, ingredient_id: 16},
+  {recipe_id: 3, ingredient_id: 17},
+  {recipe_id: 3, ingredient_id: 18},
+  {recipe_id: 3, ingredient_id: 19},
+  {recipe_id: 3, ingredient_id: 20},
+  {recipe_id: 3, ingredient_id: 21},
+  {recipe_id: 3, ingredient_id: 1},
+  {recipe_id: 4, ingredient_id: 14},
+  {recipe_id: 4, ingredient_id: 1},
+  {recipe_id: 4, ingredient_id: 22},
+  {recipe_id: 4, ingredient_id: 23},
+  {recipe_id: 4, ingredient_id: 24},
+  {recipe_id: 4, ingredient_id: 25},
+  {recipe_id: 4, ingredient_id: 26},
+  {recipe_id: 4, ingredient_id: 27},
+  {recipe_id: 4, ingredient_id: 28},
+  {recipe_id: 5, ingredient_id: 2},
+  {recipe_id: 5, ingredient_id: 29},
+  {recipe_id: 5, ingredient_id: 30},
+  {recipe_id: 5, ingredient_id: 31},
+  {recipe_id: 5, ingredient_id: 9}].each{|x| INgredientsRecipe.create(x)}
```

Comment:
Did you try to run it? `rake db:seed` ? It will not works. You have few problems here: 

INgredientsRecipe => IngredientsRecipe

you should drop `IngredientsRecipe.create` from them beginning of array